R/labels.r

```diff
@@ -19,13 +19,14 @@ update_labels <- function(p, labels) {
 #' Modify axis, legend, and plot labels
 #'
 #' Good labels are critical for making your plots accessible to a wider
-#' audience. Ensure the axis and legend labels display the full variable name.
-#' Use the plot `title` and `subtitle` to explain the main findings.
-#' It's common to use the `caption` to provide information about the
-#' data source. `tag` can be used for adding identification tags.
+#' audience. Always ensure the axis and legend labels display the full
+#' variable name. Use the plot `title` and `subtitle` to explain the
+#' main findings. It's common to use the `caption` to provide information
+#' about the data source. `tag` can be used for adding identification tags
+#' to differentiate between multiple plots.
 #'
 #' You can also set axis and legend labels in the individual scales (using
-#' the first argument, the `name`). I recommend doing that if you're
+#' the first argument, the `name`). I recommend doing this if you're
```

Comment:
Not having read through the docs as carefully as you have, I don't know the answer to this, but is use of the first-person voice common? If not, I'd switch to passive.